cmd/launch_producer/main.cpp

```diff
@@ -63,11 +64,35 @@ const char* kProducerPaths[] = {
     "/lib/" ABI "/libgpudataproducer.so",
 };
 
-std::string getDriver() {
+std::string getDriverOverridePackage() {
   std::string driver;
+  // Read gapid.driver_package_override setting to override default property.
+  FILE* file = popen("settings get global gapid.driver_package_override", "r");
+  if (file != nullptr) {
+    const int BUF_LEN = 256;
+    char driver_override[BUF_LEN];
+    if (fgets(driver_override, BUF_LEN, file) != nullptr) {
+      LOG_INFO("Driver package override: %s", driver_override);
```

Comment:
Only log this if there actually is an override.

Reply:
Done